R/getCrossAssociation.R

```diff
@@ -1150,32 +1169,32 @@ setMethod("getCrossAssociation", signature = "SummarizedExperiment",
 # Input: Correlation table and thresholds
 # Output: Filtered correlation table (or NULL if there are no observations after filtering)
 .association_filter <- function(result,
-                                p_adj_threshold,
-                                cor_threshold, 
+                                p.adj.threshold,
+                                cor.threshold, 
                                 assay1,
                                 assay2,
-                                filter_self_correlations, 
+                                filter.self.corr, 
```

Comment:
Can you still change this to filter.self.cor

That is because cor.threshold has only one "r" so good to harmonize

Reply:
ok